src/modules/footer/components/coordinateSelect/CoordinateSelect.js

```diff
@@ -0,0 +1,92 @@
+import { html } from 'lit-html';
+import { BaElement } from '../../../BaElement';
+import css from './coordinateSelect.css';
+import { $injector } from '../../../../injection';
+
+/**
+ * Dropdown for selecting the coordinate system
+ * @class 
+ * @author bakir_en  
+ */
+
+export class CoordinateSelect extends BaElement {
+
+	constructor() {
+		super();
+  
+		const { CoordinateService, EnvironmentService, MapService, TranslationService } 
+            = $injector.inject('CoordinateService', 'EnvironmentService', 'MapService', 'TranslationService');
+		this._coordinateService = CoordinateService;
+		this._environmentService = EnvironmentService;
+		this._mapService = MapService;
+		this._translationService = TranslationService;  
+	}
+    
+	/**
+     * @override 
+     */
+	isRenderingSkipped() {
+		return this._environmentService.isTouch();
+	}
+    
+	/**
+	 * @override
+	 */
+	onWindowLoad() {
+		// without this if clause,  this._view equals null
+		if (!this._environmentService.isTouch()) {
+			this._view = this.shadowRoot.querySelector('.select-coordinate');
+			this._selected = this._view.value;
+		} 
+	}
```

Comment:
Avoid holding HTML elements as local state, just data.
In this case I would suggest two fields which are initially  set within the constructor
-  `_items` (loaded by the MapService )
- `_selectedCode`  the code value of the first item

The whole creation of the View just depends on that two fields. When the select element changes, `_selectedCode` is being updated and the whole view must be re-rendered. The re-rendering has to be done manually